Codeinwp/neve
# Toggle added to enable Hosting Locally the Google Fonts

### Summary

A toggle was added to enable Google Fonts to host locally.

If the toggle is enabled; the Google Fonts should be hosted locally. Before that, we were offering a snippet to enable it. https://docs.themeisle.com/article/1349-how-to-load-neve-fonts-locally#code-snippet

From now on we have toggle + snippet (snippet overrides the toggle)
![Screen Shot 2022-11-20 at 20 35 56](https://user-images.githubusercontent.com/25361626/202917109-4800c089-1d19-44b4-b7d1-66607d0843aa.png)


### Will affect visual aspect of the product

YES

### Screenshots 

### Test instructions

- The toggle was added to Customizer -> Typography
- If the toggle is activated; Google fonts should be loaded locally.


Btw: Also, a quick look at the customizer would be good against regression on the customizer overall.


Closes #3671.
